src/Runtime/Serializer.hs

```diff
@@ -0,0 +1,48 @@
+{-# LANGUAGE DeriveDataTypeable #-}
+-- Serializer.hs
```

Comment:
Maybe we should have another subfolder for interface-related stuff? Then we could also put the server API there too, when we get to that.